src/PrestaShopBundle/Resources/config/services/core/util/number.yml

```
@@ -0,0 +1,8 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.core.util.number.number_extractor:
+    class: 'PrestaShop\PrestaShop\Core\Util\Number\NumberExtractor'
+    arguments:
+      '@property_accessor'
```

Oh that's why travis fails!
```suggestion
      - '@property_accessor'
```